### ISSUE TITLE
Make the events component functional

### ISSUE THREAD

**Issue by plusk** [dotkom/regme#70]
I suggest the buttons are basic at first, and when made active (the first one upon opening the app, or when switched to another event), has mdl-button--raised added to its classes.
Determining which event is active is probably part of some larger state logic within the app, as it reflects on several other components too.
Need to have a chat about how the state structure will work, as the event switching is one of the most important parts of the app.

**Comment by duvholt** [comment]
If you need to dynamically add classes to elements I would suggest that you use [classnames](https://www.npmjs.com/package/classnames).

``` js
var btnClass = classNames({
      'btn': true,
      'btn-pressed': this.state.isPressed,
      'btn-over': !this.state.isPressed && this.state.isHovered
});
```

**Comment by plusk** [comment]
This was fixed in #77.
